config/chime_psr_verification.yaml

```diff
@@ -127,9 +178,7 @@ gpu:
     kotekan_stage: hsaProcess
     gpu_id: 0
     updatable_gain_psr: /pulsar_gain
-    source_ra: [300.022454326,  299.772454326, 300.272454326 , 300.022454326, 300.022454326, 299.771454326 , 299.771454326 , 300.273454326,  300.273454326, 300.022454326]
-    source_dec: [40.7839418602, 40.7839418602, 40.7839418602, 41.0339418602, 40.5339418602, 40.5329418602 , 41.0349418602 , 40.5329418602 , 41.0349418602 , 40.7839418602]
-    psr_scaling: [48,48,48,48,48,48,48,48,48,48]
+    updatable_psr_pt: /pulsar_pointing
```

Comment:
I think we once decided to only use a fixed key here: `updatable_config` (https://github.com/kotekan/kotekan/blob/556a62159b0a303f652febb1f851751f64b573b9/lib/core/configUpdater.hpp#L32), but I leave it to @andrerenard if we should apply that strictly.

Reply:
This is a bit of an odd case because we have a bit of an internal framework with-in this stage, with its own subcomponents.   However in the interest of consistency, yes, let's change that to: 
```
updatable_config:
   gain_psr: /pulsar_gain
   psr_pt: /pulsar_pointing
```
And then update the `hsaCommand` objects that use those.